### PR TITLE
Add a backward compatibility test.

### PR DESCRIPTION
**Pull request by grischoun**

### CODE REVIEW

**Review by whilo — DISMISSED** [review]
Nice work.

File: test/datahike/backward_compatibility_test/backward_compatibility_test.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+
+
+# Run by being in the dir where this file is defined.
```
Comment (kordano) [inline test/datahike/backward_compatibility_test/backward_compatibility_test.sh:5]: Could you adjust that to run on the root folder so we can include that into our pipelines?

**Comment by kordano** [comment]
@TimoKramer could you have a look on how to integrate this in our pipelines?

File: test/datahike/backward_compatibility_test/src/backward_test.clj [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+              :db/valueType   :db.type/long}])
+(def cfg {:store  {:backend :file :path "/tmp/datahike-backward-comp-test"}
+          :keep-history? true
+          :schema-flexibility :write
+          :initial-tx schema})
+
+(def size 10000)
+
+(defn write [opt]
+  (println "----- Writing to db using latest released version of datahike ....")
```
[the remark this answers is not in the document]
Comment (grischoun) [reply test/datahike/backward_compatibility_test/src/backward_test.clj]: Done.

File: test/datahike/backward_compatibility_test/backward_compatibility_test.sh (hunk 1 above)
Comment (grischoun) [reply test/datahike/backward_compatibility_test/backward_compatibility_test.sh:5]: Done.

**Comment by kordano** [comment]
The CI is set up so we can merge this now?

**Comment by grischoun** [comment]
> The CI is set up so we can merge this now?

From my side, yes.

**Review by kordano — APPROVED** [review]

**Pull request merged**